### PR TITLE
Peer role fix using syncsource

### PR DESCRIPTION
**Pull request by fboucquez**

### CODE REVIEW

File: src/service/ConfigLoader.ts [hunk 1]
```diff
@@ -371,62 +380,60 @@ export class ConfigLoader {
...
+                enableSingleThreadPool: false,
+                addressextraction: true,
+                mongo: true,
+                zeromq: true,
+                enableAutoSyncCleanup: false,
+            };
+        }
+        //peer only (harvesting or not).
+        return {
+            sinkType: 'Sync',
```
Comment (Wayonb) [inline src/service/ConfigLoader.ts:419]: The default sinkType should be Async

File: src/service/ConfigLoader.ts [hunk 2]
```diff
@@ -371,62 +380,60 @@ export class ConfigLoader {
...
+            enableSingleThreadPool: true,
```
Comment (Wayonb) [inline src/service/ConfigLoader.ts:420]: enableSingleThreadPool should be false

**Review by Wayonb — CHANGES_REQUESTED** [review]

File: src/service/ConfigLoader.ts (hunk 1 above)
Comment (fboucquez) [reply src/service/ConfigLoader.ts:419]: Both changes are unrelated to the fix in this PR. 
syncType: Sync and enableSingleThreadPool: true has been there for a long time for peer-only nodes. Are we sure we want to apply this change now?

Comment (Wayonb) [reply src/service/ConfigLoader.ts:419]: opened https://github.com/nemtech/symbol-bootstrap/issues/217

**Review by Wayonb — APPROVED** [review]

**Pull request merged**